### ISSUE TITLE
Tp-Link Deco DDNS doesn't update

### ISSUE THREAD

**Issue by chinmaythosar** [nextdns/nextdns#728]
Hardware - Deco X20 v4

Decos (at least x20) have a tplink ddns setup available where what I do is setup the first part of the url as the config ID of nextdns and then the tplink url. So I get something like abcd1234.tplinkdns.com . I can enable ping from wan (not recommended) and I can ping the ddns so I know it works. However when I set that DDNS in my config in nextdns it seems to not update the IP address. Not exactly sure if its an ISP issue but the IP I can see in NextDNS isnt the same as the one that my deco seems to get. Could be a deco firmware issue or my ISP but was wondering if anyone can look into this if they have a Deco.

**Comment by chinmaythosar** [comment]
Ignore. There was an issue with tplink itself. It seems to behave unusually sometimes. I was able to register it and works now.

**Comment by chinmaythosar** [comment]
Retested it. Turns out sometimes the configuration in nextdns seems to fail to update the IP using DDNS. It mentions every minute but it doesn’t seem to be doing it every minute. As a result sometimes I even get an error saying the IPs differ if I try to add the ddns. I think it’s getting the wrong IP somehow from the link. I was able to get the correct IP from the ddns link as well as from the nextdns config page but whatever script is being run in the background every minute by the config seems to not be able to correctly get the IP.
